src/dist/replication/meta_server/greedy_load_balancer.cpp

```diff
@@ -945,5 +964,56 @@ void greedy_load_balancer::report(const dsn::replication::migration_list &list,
         _recent_balance_copy_secondary_count->add(counters[COPY_SEC_COUNT]);
     }
 }
+
+std::string
+greedy_load_balancer::set_balancer_ignored_app_id_list(const std::vector<std::string> &args)
+{
+    std::string invalid_arguments("invalid arguments");
+    std::stringstream oss;
+    if (args.empty()) {
+        dsn::zauto_read_lock l(_balancer_ignored_apps_lock);
+        oss << "ignored_app_id_list: ";
+        std::copy(_balancer_ignored_apps.begin(),
+                  _balancer_ignored_apps.end(),
+                  std::ostream_iterator<dsn::replication::app_id>(oss, ","));
+        std::string app_ids = oss.str();
+        app_ids.erase(app_ids.size() - 1);
```

Comment:
erase may be heavy, because internally it may be implemented by memove. 
```suggestion
        app_ids[app_ids.size() - 1] = '\0';
```